packages/acgtk/acgtk.1.5.1/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "sylvain.pogodalla@inria.fr"
+
+build: [
+  ["dune" "subst"] {pinned}
+# remove the -p to also build the local libraries: conflict with the
+# fact that some libraries are also part of the acgtkLib package
+#  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "-j" jobs]
+]
+
+install: ["dune" "install"]
```

Comment:
```suggestion
```
Dune already support installs using the `<package>.install` file that opam uses.

Reply:
oh right nevermind, I should have seen the comment above